### PR TITLE
Update service definition docs

### PR DESCRIPTION
**Pull request by simon-johansson**

### CODE REVIEW

File: docs/ts/primitives/services.mdx [hunk 1]
```diff
@@ -25,4 +25,29 @@ export default new Service("my-service");
 
 That's it! Encore will consider this directory and all its subdirectories as part of the service.
 
-For more on how to structure your application, see the [app structure guide](/docs/ts/develop/app-structure).
\ No newline at end of file
+For more on how to structure your application, see the [app structure guide](/docs/ts/develop/app-structure).
+
+## Defining services implicitly
```
Comment (simon-johansson) [inline docs/ts/primitives/services.mdx:30]: I think we should describe this way of creating services as well as current users probably have this setup. We might also have some examples floating around defining services in this way. Thoughts?

File: docs/quick-start.mdx [hunk 2]
```diff
@@ -289,10 +289,17 @@ interface Response {
...
-If you want to create more services and endpoints, you simply create new folders and define endpoints by wrapping functions in the `api` function. _If you're curious, you can read more about defining [services](/docs/ts/primitives/services) and [APIs](/docs/ts/primitives/apis)._
+The `world` endpoint is part of the `hello` service because in the same folder you will also find a file named `encore.service.ts` which looks like this:
+
+```ts
+import { Service } from "encore.dev/service";
+
+export default new Service("hello");
+```
+
+This is how you create microservices with Encore. Encore will now consider files in the `hello` directory and all its subdirectories as part of the `hello` service. If you want to create more services, simply create a new folders, add a `encore.service.ts` file that is exporting a new `Service`. _If you're curious, you can read more about defining [services](/docs/ts/primitives/services) and [APIs](/docs/ts/primitives/apis)._
```
Comment (eandre) [inline docs/quick-start.mdx:302]:
```suggestion
This is how you define services with Encore. Encore will now consider files in the `hello` directory and all its subdirectories as part of the `hello` service. If you want to create more services, simply create a new folders, add a `encore.service.ts` file that is exporting a new `Service`. _If you're curious, you can read more about defining [services](/docs/ts/primitives/services) and [APIs](/docs/ts/primitives/apis)._
```

File: docs/tutorials/rest-api.mdx [hunk 3]
```diff
@@ -377,9 +377,29 @@ This is needed when you want Encore to manage functionality like secrets and han
...
+🥐 Add the following code to `url/encore.service.ts`:
+
+```ts
+-- url/encore.service.ts --
+import { Service } from "encore.dev/service";
+
+export default new Service("url");
+```
+
+This is how you create microservices with Encore. Encore will now consider files in the `url` directory and all its subdirectories as part of the `url` service.
```
Comment (eandre) [inline docs/tutorials/rest-api.mdx:396]:
```suggestion
This is how you define a service with Encore. Encore will now consider files in the `url` directory and all its subdirectories as part of the `url` service.
```

**Review by marcuskohlberg — DISMISSED** [review]

File: docs/tutorials/slack-bot.md [hunk 4]
```diff
@@ -373,7 +373,25 @@ Since Slack sends custom HTTP headers that we need to pay attention to, we're go
...
+🥐 Add the following code to `slack/encore.service.ts`:
+
+```ts
+-- slack/encore.service.ts --
+import { Service } from "encore.dev/service";
+
+export default new Service("slack");
+```
+
+This is how you create microservices with Encore. Encore will now consider files in the `slack` directory and all its subdirectories as part of the `slack` service.
```
Comment (eandre) [inline docs/tutorials/slack-bot.md:392]:
```suggestion
This is how you create define services with Encore. Encore will now consider files in the `slack` directory and all its subdirectories as part of the `slack` service.
```

File: docs/tutorials/uptime.md [hunk 5]
```diff
@@ -912,13 +912,26 @@ Let's start by creating the functionality to check if a website is currently up
...
+🥐 Add the following code to `monitor/encore.service.ts`:
+
+```ts
+-- monitor/encore.service.ts --
+import { Service } from "encore.dev/service";
+
+export default new Service("monitor");
+```
+
+This is how you create microservices with Encore. Encore will now consider files in the `monitor` directory and all its subdirectories as part of the `monitor` service.
```
Comment (eandre) [inline docs/tutorials/uptime.md:931]:
```suggestion
This is how you create define services with Encore. Encore will now consider files in the `monitor` directory and all its subdirectories as part of the `monitor` service.
```

**Review by marcuskohlberg — APPROVED** [review]

**Review by eandre — APPROVED** [review]

**Pull request merged**